### PR TITLE
Glidera new buy and sell flow

### LINKED ISSUE #5136: Glidera view initialization is ugly
When entering the Glidera view the "welcome/connect" content is briefly displayed and then replaced with the buy/sell options for my logged in account.  I Need to prevent the "welcome/connect" content from showing.

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fixes #5136
fixes https://github.com/bitpay/copay/issues/4877

### CODE REVIEW

File: src/js/controllers/confirm.js [hunk 1]
```diff
@@ -422,6 +433,47 @@ angular.module('copayApp.controllers').controller('confirmController', function(
       return setSendError(gettextCatalog.getString('No wallet selected'));
     }
 
+    if ($scope.isGlidera) {
+      $scope.get2faCode(function(err, sent) {
+        if (err) {
+          popupService.showAlert(gettextCatalog.getString('Error'), gettextCatalog.getString('Could not send confirmation code to your phone'));
```
Comment (JDonadio) [inline src/js/controllers/confirm.js]: External services should not be translated. There is a lot of lines using **gettextCatalog** service. Please check and remove all of them.

File: src/js/controllers/confirm.js [hunk 2]
```diff
@@ -518,11 +571,176 @@ angular.module('copayApp.controllers').controller('confirmController', function(
...
+      $state.go('tabs.home').then(function() {
+        $state.transitionTo('tabs.buyandsell.glidera');
+      });
     } else {
       $state.go('tabs.send');
     }
   };
 
+  $scope.get2faCode = function(cb) {
+    ongoingProcess.set('Sending 2FA code...', true);
```
Comment (JDonadio) [inline src/js/controllers/confirm.js]: This text should be added to _ongoingProcess service_ (change this for all calls)

File: src/js/services/glideraService.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ angular.module('copayApp.services').factory('glideraService', function($http, $l
      * Development: 'testnet'
      * Production: 'livenet'
      */
-    credentials.NETWORK = 'livenet';
+    credentials.NETWORK = 'testnet';
```
Comment (JDonadio) [inline src/js/services/glideraService.js]: Only testnet?

File: www/views/confirm.html [hunk 4]
```diff
@@ -83,15 +116,18 @@
     </div>
   </ion-content>
   <click-to-accept
-    ng-click="approve(statusChangeHandler)"
+    ng-click="wallet ? approve(statusChangeHandler) : null"
```
Comment (JDonadio) [inline www/views/confirm.html]: Logic should be within _approve() function_

File: www/views/confirm.html [hunk 5]
```diff
@@ -83,15 +116,18 @@
...
+    click-send-status="sendStatus"
+    has-wallet-chosen="wallet">
     Click to pay
   </click-to-accept>
   <slide-to-accept
+    ng-disabled="!wallet"
     ng-if="isCordova && wallets[0] && !insufficientFunds && !noMatchingWallet"
-    slide-on-confirm="onConfirm()"
-    slide-send-status="sendStatus">
+    slide-on-confirm="wallet ? onConfirm() : null"
```
Comment (JDonadio) [inline www/views/confirm.html]: Same as above

**Comment by JDonadio** [comment]
LGTM

**Comment by JDonadio** [comment]
ACK

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**